modules/core-services/endpoints/member.go

```diff
@@ -148,7 +148,8 @@ func (e *Endpoints) ListMemberRoles(ctx context.Context, r *http.Request, vars m
 			}
 		}
 	}
-	if orgID == 0 {
+
+	if orgID == 0 && scopeType != apistructs.SysScope {
```

Comment:
different condition should return different error

Reply:
I think it's the same condition that means the orgID can't be equal to 0, only if the scopeType is 'sys'.